go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -1179,49 +1179,13 @@ func main() {
 	udp := flag.Bool("udp", false, "use udp transport protocol")
 	addressFamily := flag.String("addr-family", ip4, "Address family (ip4 or ip6) to be used for tracing")
 	skipOvnDetrace := flag.Bool("skip-detrace", false, "skip ovn-detrace command")
+	dumpUserDefinedNetworksVRFTableIDsForNetwork := flag.String("dump-udn-vrf-table-ids", "", "Dump the VRF table ID per node for specified user defined network")
```

Comment:
> nit: its too long (the name maybe just `dumpUDNVRFIDs`) ? for network is understood since you have UDN also in future if we want to do the same for finding ovn allocated zones, marks, we might want to make this more extensible?
> 

I will rename into `dumpVRFTableIDs` 

> so if we go with `--dump-udn-vrf-table-ids` it dumps the map right?: networka: 10 networkb: 15
> 
> like that? or are you taking network name as input and then dumping the vrfid for that network across all nodes in the cluster? could you please add some more details to your commit message?

VRF Table IDs are going to be different per Node since the interface index is something decided by the kernel, so this is going to return `{node1: 10 node2: 15}` for the network specified

I am going to just print all the networks and move the flag as a boolean.

Reply:
yes I like that better, let's print all networks! also let's call out that this command is expected to output the results of a specific node right? its not that we plan to take nodeName as a parameter?

Reply:
It will print it for all the nodes, users can filter by node the output using `jq` or the like.